corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -161,7 +162,13 @@ hqDefine('registration/js/new_user.ko', [
                 zxcvbnPassword: 2,
             });
 
-
+        self.captcha = ko.observable(defaults.captcha)
+            .extend({
+                required: {
+                    message: django.gettext("Please enter captcha."),
```

Comment:
You can use just `gettext` to avoid this warning, since it's configured as an allowed global: https://github.com/dimagi/commcare-hq/blob/9326fe322017258f4c926841e977d87b085145de/.eslintrc.js#L18

Reply:
@orangejenny i am not able to retreive captcha data into submit data. need some help on that

Reply:
What is the captcha data supposed to be? When I look at other places that use the captcha ([password reset form](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/registration/static/registration/js/password_reset_form.js) and [accepting invitations](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/users/static/users/js/accept_invite.js)), they don't do anything in js, they just include the `hqwebapp/js/captcha` module.

Reply:
here we have helper layout defined. so needed to include fields in the js too (that is my assumption).